src/higley_lab_to_nwb/benisty_2024/interfaces/benisty_2024_cidansegmentation_interface.py

```diff
@@ -0,0 +1,131 @@
+from typing import Optional
+from pynwb import NWBFile
+from copy import deepcopy
+
+
+from neuroconv.datainterfaces.ophys.basesegmentationextractorinterface import BaseSegmentationExtractorInterface
+from neuroconv.utils import DeepDict, FilePathType
+
+from .benisty_2024_cidansegmentation_extractor import (
+    Benisty2024CidanSegmentationExtractor,
+)
+
+def format_string_for_parameters_dict(dict_obj, indent=2):
+    def format_key_value(key, value, level):
+        indentation = ' ' * (level * indent)
+        if isinstance(value, dict):
+            formatted_value = format_dict(value, level + 1)
+            return f"{indentation}{key}:\n{formatted_value}"
+        else:
+            return f"{indentation}{key}: {value}\n"
+    
+    def format_dict(d, level):
+        return ''.join([format_key_value(k, v, level) for k, v in d.items()])
+    
+    return format_dict(dict_obj, 0)
+
+class Benisty2024CidanSegmentationInterface(BaseSegmentationExtractorInterface):
+    """Interface for Suite2p segmentation data."""
+
+    display_name = "CIDAN Segmentation"
+    associated_suffixes = (".json", ".mat")
+    info = "Interface for CIDAN segmentation."
+    Extractor = Benisty2024CidanSegmentationExtractor
+
+    def __init__(
+        self,
+        parameters_file_path: FilePathType,
+        roi_list_file_path: FilePathType,
+        mat_file_path: FilePathType,
+        sampling_frequency: float,
+        plane_segmentation_name: Optional[str] = None,
+        verbose: bool = True,
+    ):
+        """Create SegmentationExtractor object out of CIDAN data type.
+
+        Parameters
+        ----------
+        parameters_file_path: str or Path
+            The path to the CIDAN parameter .json file.
+        roi_list_file_path: str or Path
+            The path to the CIDAN roi_list .json file.
+        mat_file_path: str or Path
+            The path to the CIDAN df/f traces .mat file.
+        sampling_frequency: float
+            The sampling frequency of the fluorescence traces
+        plane_segmentation_name: str, optional
+            The name of the plane segmentation to be added.
+        """
+
+        super().__init__(
+            parameters_file_path=parameters_file_path,
+            roi_list_file_path=roi_list_file_path,
+            mat_file_path=mat_file_path,
+            sampling_frequency=sampling_frequency,
+        )
+
+        if plane_segmentation_name is None:
+            plane_segmentation_name = "PlaneSegmentation"
```

Comment:
You can flatten this with:
```suggestion
       plane_segmentation_name = plane_segmentation_name or "PlaneSegmentation"
```